test/WebSites/FormatFilterWebSite/Controllers/ProducesOverrideClass.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc;
+
+namespace FormatFilterWebSite
+{
+    [Produces("application/custom_ProducesOverrideController")]
+    public class ProducesOverrideController : ProducesBaseController
```

Comment:
Isn't that what you meant that I should add when you said overriding produces test case? This is what we talked about. Looks like I understood wrong...?